packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -160,7 +171,7 @@ class _AndroidViewState extends State<AndroidView> {
     return _AndroidPlatformView(
         controller: _controller,
         hitTestBehavior: widget.hitTestBehavior,
-        gestureRecognizers: widget.gestureRecognizers,
+        gestureRecognizers: widget.gestureRecognizers ?? Set<Factory<OneSequenceGestureRecognizer>>(),
```

Comment:
there's a lot of places in this PR where you either have defaults, or default from null, or whatever (sometimes with a constructor like here, sometimes with a precreated static variable, etc). It would be nice if there was only one place that needed to default, and on one side of that null was considered acceptable, and on the other side it was not.

Reply:
Left this the only place that defaults for null, the classes downstream now require a non null value.